show/reboot_cause.py

```diff
@@ -26,18 +30,21 @@ def read_reboot_cause_file():
 def reboot_cause(ctx):
     """Show cause of most recent reboot"""
     if ctx.invoked_subcommand is None:
-        reboot_cause = ""
-        # Read the previous reboot cause
-        data = read_reboot_cause_file()
-        if data['user'] == "N/A":
-            if data['cause'] == "Kernel Panic":
-                reboot_cause = "{} (Time: {})".format(data['cause'], data['time'])
-            else: 
-                reboot_cause = "{}".format(data['cause'])
-        else:
-            reboot_cause = USER_ISSUED_REBOOT_CAUSE_REGEX.format(data['cause'], data['user'], data['time'])
+        reboot_cause_str = ""
+
+        # Read the previous reboot reason
+        reboot_cause = read_reboot_cause_file()
+
+        if reboot_cause and "cause" in reboot_cause.keys():
+            reboot_cause_str = "Cause: {}".format(reboot_cause["cause"])
+
+            if "user" in reboot_cause.keys() and reboot_cause["user"] != "N/A":
+                reboot_cause_str += ", User: {}".format(reboot_cause["user"])
+
+            if "time" in reboot_cause.keys() and reboot_cause["time"] != "N/A":
+                reboot_cause_str += ", Time: {}".format(reboot_cause["time"])
```

Comment:
No need for "Cause:` prefix. Also, we lose the "User issued '<command>' command" syntax here. Instead we just see "<command>".

Suggest keeping format the same as before.